tests/test.rre.js

```diff
@@ -93,7 +93,10 @@ describe('RRE Decoder', function () {
         display.fillRect(2, 0, 2, 2, [ 0x00, 0xff, 0x00 ]);
         display.fillRect(0, 2, 2, 2, [ 0x00, 0xff, 0x00 ]);
 
-        testDecodeRect(decoder, 1, 2, 0, 0, [ 0x00, 0xff, 0xff, 0xff, 0xff ], display, 24);
+        testDecodeRect(decoder, 1, 2, 0, 0,
+                       [ 0x00, 0x00, 0x00, 0x00,
+                         0xff, 0xff, 0xff, 0xff ],
+                       display, 24);
```

Comment:
It's really a flaw in the tests. The success condition is that nothing happens. So this test succeeded because the decoder wanted more data, not because it was done processing and found nothing to render.

I'll have a look if that distinction can be detected in the tests.